scratch/shawn/scripts/shawnhammer.sh

```diff
@@ -129,6 +129,13 @@ if [ "$?" = "1" ]; then
     docker rm -f $(docker ps -q -f name=pysmurf | awk '{print $1}')
 fi
 
+# stop all smurf-streamer dockers
+matching_dockers smurf-streamer
+if [ "$?" = "1" ]; then
+    echo "-> Stopping all running stable pysmurf dockers."
```

Comment:
Please differentiate the echo statement from the preceeding.  Maybe "Stopping all running smurf-streamer dockers"?